scripts/GCE_GPU_Startup.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+echo "SUDO_USER =" $SUDO_USER
+
+echo "Adding graphics drivers repo."
+sudo add-apt-repository -y ppa:graphics-drivers/ppa
```

Comment:
Can we remove all these echos? I think if we do something [like this](https://stackoverflow.com/questions/5750450/print-each-command-before-executing) we can not need that?

Reply:
Good call. I'll make that change.